update_version.sh

```diff
@@ -111,6 +112,7 @@ fi
 SED_STR_VERSION="s/${VERSION}/${NEW_VERSION}/g"
 SED_STR_PACKAGE="s/<version>${VERSION}<\/version>/<version>${NEW_VERSION}<\/version>/g"
 SED_STR_DOXYGEN="s/PROJECT_NUMBER = ${VERSION}/PROJECT_NUMBER = ${NEW_VERSION}/g"
+SED_STR_TOML="s/version = \"${VERSION}\"/version = \"${NEW_VERSION}\"/g"
```

Comment:
not restrictive enough, this has the potential of replacing version constraints of dependencies in the package manifest. The control libraries dependency is currently default constrained, but we might add constraints in future. In that case, if modulo version ever happens to match the CL we might have an awkward bug that we accidentally update both and don't notice.

For example:
```
[metadata]
version = "6.2.0"

[build.dependencies]
"@aica/foss/control-libraries" = {version = "v6.2.0", constraints = ">= 6.0.0"}
```

Reply:
Actually it's safe to ignore the above, since I realized the dependency version tag includes `v` and the metadata version does not.